app/src/main/java/id/rizmaulana/covid19/ui/detail/DetailViewModel.kt

```diff
@@ -76,4 +76,15 @@ class DetailViewModel(
             .addTo(compositeDisposable)
     }
 
+    fun putPrefCountry(data: CovidDetail) {
+        appRepository.putPrefCountry(data)
+            .subscribeOn(schedulerProvider.ui())
+            .subscribe({
+                errorMessage.postValue("Success")
```

Comment:
`errorMessage` for a success message?

Reply:
basically its a toast message, let me rename to improve the meaning